README.md

```diff
@@ -214,6 +214,21 @@ The [`python-path` output value][`python-path`] of the [setup-python] action
 The actual value of `origin-python-version` passed to the [setup-python] action
 
 
+### `sanity-tests`
+
+Comma-separated list of sanity tests to run. If not present, all applicable tests are run.
+
+
+### `sanity-skip-tests`
+
+Comma-separated list of sanity tests to skip.
+
+
+### `sanity-allow-disabled`
+
+Allow sanity tests to run which are disabled by default.
```

Comment:
How about reusing the default value style used at the top of the inputs list in this document?
```suggestion
Allow running sanity tests that are disabled by default.
**(DEFAULT: `false`)**
```